hoomd/MemoryTraceback.cc

```diff
@@ -90,6 +98,7 @@ void MemoryTraceback::outputTraces(std::shared_ptr<Messenger> msg) const
     msg->notice(2) << "Actual allocation sizes may be larger by up to the OS page size due to alignment." << std::endl;
     msg->notice(2) << "List of memory allocations and last " << MAX_TRACEBACK-1 << " functions called at time of (re-)allocation" << std::endl;
 
+#ifdef __GNUC__
```

Comment:
For features that specifically break on Windows, I would prefer to detect “not Windows” instead of `__GNUC__`. I do not know the full compiler and platform compatibility of `__GNUC__` but I suspect there may be compilers that work with HOOMD and do not have that definition.